Makefile

```diff
@@ -73,6 +73,11 @@ PD_SERVER_DEP += dashboard-ui
 pd-server: ${PD_SERVER_DEP}
 	CGO_ENABLED=$(BUILD_CGO_ENABLED) go build $(BUILD_FLAGS) -gcflags '$(GCFLAGS)' -ldflags '$(LDFLAGS)' -tags "$(BUILD_TAGS)" -o $(BUILD_BIN_PATH)/pd-server cmd/pd-server/main.go
 
+pd-server-failpoint:
+	@$(FAILPOINT_ENABLE)
+	$(MAKE) pd-server BUILD_TAGS="$(BUILD_TAGS) with_fail"
```

Comment:
What about 
```
ifeq ($(FAILPOINT), 1)
	BUILD_TAGS += with_fail
endif
```
Then use `FAILPOINT=1 $(MAKE) pd-server`

Reply:
It should make `failpoint-enale` first.

Reply:
It doesn't conflict?

Reply:
I think the current method is cleaner than the add some logic about if check and enable/disable failpoint rewrite in the `pd-server` command.

Reply:
oh, I get your point.